src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import BooksList from '../containers/BooksList';
+import BooksForm from '../containers/BooksForm';
+
+const App = () => (
+  <>
+    <div className="booksList"><BooksList /></div>
+    <div className="booksForm"><BooksForm /></div>
```

Comment:
- [ ] Has children components wrapped in `<div>` tags
      [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/App/AppWrapBookListAndBookForm.png) (These components should be wrapped in just 1 div, this is a little confusing but the screenshot is a little clearer)

Reply:
Moved components into the same single div element.